specs/core/1_custody-game.md

```diff
@@ -477,7 +491,8 @@ def process_early_derived_secret_reveal(state: BeaconState, reveal: EarlyDerived
 
     assert reveal.epoch >= get_current_epoch(state) + RANDAO_PENALTY_EPOCHS
     assert reveal.epoch < get_current_epoch(state) + EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS
-    assert not revealed_validator.slashed
+    assert is_slashable_validator(revealed_validator, get_current_epoch(state))
+    assert get_current_epoch(state) < revealed_validator.exit_epoch 
```

Comment:
This is a change: We need to give the validator the opportunity to reveal their current custody secret after exit_epoch, so that they can get into withdrawable state. So from exit_epoch it is not an early reveal anymore.